apps/api/src/caching/helpers.ts

```diff
@@ -1,8 +1,11 @@
 import * as Sentry from "@sentry/node";
 import { differenceInSeconds } from "date-fns";
 
+import { LoggerService } from "@src/core";
 import MemoryCacheEngine from "./memoryCacheEngine";
 
+const logger = new LoggerService({ context: "CachingHelpers" });
```

Comment:
I'm not too hard on this, but eventually this would look better on logs :)

```suggestion
const logger = new LoggerService({ context: "Caching" });
```